docs/en/pid-data-model.rst

```diff
@@ -0,0 +1,393 @@
+
+.. include:: ../common/common_definitions.rst
+
+.. _pid_data_model.rst:
+
+PID Data Model
+++++++++++++++
+
+The Person Identification Data (PID) is issued by the PID Provider following national laws and allows a natural person to be authenitcated and identified. 
+
+The User attributes carried in the Italian PID are:
+
+    - Current Family Name
+    - Current First Name
+    - Date of Birth
+    - Place of Birth
+    - Unique Identifier
+    - Taxpayer identification number
```

Comment:
according to pid rulebook, do you agree to include also `nationalities` just for interop compatibility?

Reply:
I have some concerns regarding the `nationalities`:
- It is the only parameter that is an array, this introduces some additional difficulties in the SD-JWT handling
- It is a dynamic value, i.e. I could add a new nationality or lose it and this has an impact on the validity of the PID. 

I propose two alternatives:
1. consider `nationality` instead of `nationalities`, A value that means that the user has that nationality. for example, an user having both French and Italian nationalities, will have both French and Italian PID with `italian` or `french` as values in the `nationality` field
2. The nationalities information in a separated (Q)EAA, so that the changes in it don't affect the validity of the PID.

Reply:
@peppelinux in the new version of PID rulebook the `nationality` field is optional. As discussed we decide to not consider it in the PID mandatory dataset